pkg/client/redis/client.go

```diff
@@ -272,6 +272,10 @@ func (c *client) SetCustomRedisConfig(ip string, configs map[string]string, auth
 	rClient := rediscli.NewClient(options)
 	defer rClient.Close()
 
+	if err := c.applyRedisConfig("replica-announce-ip", ip, rClient); err != nil {
```

Comment:
There is no `replica-announce-ip` configuration in redis 3.2.2, I hope the operator supports redis 3.2.2+.